### DISCUSSION TITLE
Rebase error; Anyone has seen similar problems?

### DISCUSSION THREAD

**Discussion by XuhuiZhou** [alshedivat/al-folio#312] [Q&A]
Following the README:
```
$ git remote add upstream https://github.com/alshedivat/al-folio.git
$ git fetch upstream
$ git rebase upstream/v0.3.5
```
However, no matter

`git rebase upstream/v0.3.6`
or
`git rebase upstream/v0.3.5`

produce error:
`fatal: Needed a single revision invalid upstream 'upstream/upstream/v0.3.5'`

Additional context
I also tried to rebase in both `source` and `master` branch, but they do not work.

**Comment by alshedivat** [comment]
Haven't seen this issue before. What happens if you try `git pull --rebase upstream/v0.3.6`?

**Reply by XuhuiZhou** [reply]
Ooh, it works now. Thanks!
